definitions/npm/@ginkgo-bioworks/react-json-schema-form-builder_v1.x.x/test_react-json-schema-form-builder_v1.x.x.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React from 'react';
+import {
+  FormBuilder,
+  PredefinedGallery,
+} from '@ginkgo-bioworks/react-json-schema-form-builder';
+import { it, describe, expect } from 'flow-typed-test';
+import { mount } from 'enzyme';
+
+describe('@ginkgo-bioworks/react-json-schema-form-builder', () => {
+  const props = {
+    schema: '',
+    uischema: '',
+    onChange: (newSchema, newUiSchema) => {},
+  };
+  it('render form builder', () => {
+    const wrapper = mount(<FormBuilder {...props} />);
+    expect(wrapper.exists('.form-body')).toBeTruthy();
```

Comment:
The tests are to assert type safety, not behavior of the library. As such, you shouldn't be importing anything other than the library under test and the desc/it imports (basically).

Feel free to look at some other libdefs for examples. 😄

Reply:
thanks for the heads up! added a couple more tests that just check for a flow error when expected